File: projectEuler/2/2.py
def sum_of_even_fibonacci(limit):
    a, b = 1, 2
    total = 0
    while a <= limit:
        if a % 2 == 0:
            total += a
        a, b = b, a + b
    return total

result = sum_of_even_fibonacci(4000000)
print(f"Sum of even-valued terms in Fibonacci sequence below 4 million: {result}")
#4613732

File: projectEuler/2/2.cs
using System;

class Program
{
    static void Main()
    {
        int limit = 4000000;
        int a = 1, b = 2;
        int sum = 0;

        while (a <= limit)
        {
            if (a % 2 == 0)
            {
                sum += a;
            }
            int next = a + b;
            a = b;
            b = next;
        }

        Console.WriteLine("Sum of even-valued terms in Fibonacci sequence below 4 million: " + sum);
    }
}